package.json

```diff
@@ -2,7 +2,6 @@
   "name": "react-antd-todo",
   "version": "0.1.0",
   "private": true,
-  "homepage": "https://leonardopliski.github.io/react-antd-todo",
```

Comment:
this causes problems for serving files https://stackoverflow.com/questions/54340240/create-react-app-build-uncaught-syntaxerror-unexpected-token

Reply:
Thanks for your contribution. QA Wolf is awesome!